lib/packages/docutils/rstgen.nim

```diff
@@ -57,6 +57,8 @@ type
     options*: RstParseOptions
     findFile*: FindFileHandler
     msgHandler*: MsgHandler
+    destBaseDir*: AbsoluteDir
```

Comment:
is that different form `thisDir` ?

Reply:
- `thisDir` is the directory containing the nested Nim file. I am mapping `d.conf.outDir` from `docgen` to the new `d.destDir` in rstgen
- Also rstgen cannot "see" `thisDir`

Reply:
May be I should refactor destBaseDir to outDir.. it's clear that way.